pyproject.toml

```diff
@@ -27,7 +27,9 @@ types-requests = "^2.28.11.7"
 mimesis = "^7.0.0"
 requests-mock = "^1.10.0"
 pandas-stubs = "^2.0.0"
-black = "^23.3.0"
```

Comment:
was there a merge conflict? I think that
- you removed black
- the [tool.black] must be at the end. this is poetry/tomlkit bug and we can't mix sections...